homeassistant/components/cover/mqtt.py

```diff
@@ -44,6 +57,16 @@
     vol.Optional(CONF_STATE_OPEN, default=STATE_OPEN): cv.string,
     vol.Optional(CONF_STATE_CLOSED, default=STATE_CLOSED): cv.string,
     vol.Optional(CONF_OPTIMISTIC, default=DEFAULT_OPTIMISTIC): cv.boolean,
+    vol.Optional(CONF_TILT_CLOSED_POSITION,
+                 default=DEFAULT_TILT_CLOSED_POSITION): cv.string,
+    vol.Optional(CONF_TILT_OPEN_POSITION,
+                 default=DEFAULT_TILT_OPEN_POSITION): cv.string,
+    vol.Optional(CONF_DEVICE_CLASS,
+                 default=DEFAULT_DEVICE_CLASS): cv.string,
+    vol.Optional(CONF_TILT_MIN,
+                 default=DEFAULT_TILT_MIN): cv.string,
```

Comment:
My blinds report 0-180. They also travel from fully tilted down to fully tilted up so "open" isn't at one extreme or the other.